app/javascript/components/search/results/StudySearchResult.js

```diff
@@ -182,7 +182,8 @@ export default function StudySearchResult({ study }) {
       <div key={study.accession}>
         <label htmlFor={study.name} id="result-title" className="study-label">
           {study.study_source === 'SCP' ? <a href={study.study_url} dangerouslySetInnerHTML={displayStudyTitle} ></a> :
-            <span dangerouslySetInnerHTML={displayStudyTitle} />
```

Comment:
This is now a little long for a ternary construct for my taste.   So maybe something up above like
```
let studyLink = <a href={study.study_url} dangerouslySetInnerHTML={displayStudyTitle} ></a>
if (study.study_source !== 'SCP') {
  studyLink = <a href={`https://data.humancellatlas.org/ex...
}
```